### DISCUSSION TITLE
Can we possibly do more with history in the CLI?

### DISCUSSION THREAD

**Discussion by jaybird110127** [llm-workflow-engine/llm-workflow-engine#145] [Ideas]
We have !history to review our most recent 20 chats, the !delete command to remove one, and the !title command to give the current chat a title. Any chance of doing any of the following?

- Retrieving more than 20 history entries? The website itself can do this if you hit the Show More button.
- - Print to the screen and/or log to the currently opened log file the conversation history for a given history entry?
- - Switch context to presumably the most recent navpoint in a conversation by using the number of the history entry rather than the UUID?
- - Batch retrieving conversation histories from all or a portion of chat history, and saving each history to its own log file?
Just a few ideas.

**Comment by thehunmonkgroup** [comment]
I think most or all of these have been implemented now.
